packages/wrangler/src/api/startDevWorker/LocalRuntimeController.ts

```diff
@@ -151,6 +151,7 @@ export class LocalRuntimeController extends RuntimeController {
 					await convertToConfigBundle(data),
 					this.#proxyToUserWorkerAuthenticationSecret
 				);
+			options.liveReload = false; // TODO: set in buildMiniflareOptions once old code path is removed
```

Comment:
Issue to track this?

Reply:
Don't think that's necessary. It's just a refactor. No behaviour difference or heavy uplift.